majordodo-core/src/test/java/majordodo/client/HttpClientLeaderRecoveryTest.java

```diff
@@ -26,10 +26,14 @@
 import java.util.logging.ConsoleHandler;
 import java.util.logging.Level;
 import java.util.logging.SimpleFormatter;
-import jakarta.servlet.ServletException;
-import jakarta.servlet.http.HttpServlet;
-import jakarta.servlet.http.HttpServletRequest;
-import jakarta.servlet.http.HttpServletResponse;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import majordodo.client.ClientConnection;
```

Comment:
Why add this imports? I don't see relative code changes

Reply:
came from the revert, cleaned